spec/controllers/media_objects_controller_spec.rb

```diff
@@ -16,6 +16,7 @@
 
 describe MediaObjectsController, type: :controller do
   include ActiveJob::TestHelper
+  include ActiveSupport::Testing::TimeHelpers
```

Comment:
I think this isn't needed anymore with your last commit.
```suggestion
```

Reply:
Good catch